### ISSUE TITLE
Path on windows bug due to string template

### ISSUE THREAD

**Issue by Sn0wFox** [vitaliy-bobrov/angular-hot-loader#13]
Hi !
I have a project where we have to use Angular 1.x, and I thought that it could be really nice to have some hmr like in 2.x, so i came across your loader.

I'm working on windows, and I had an error like this:
```
ERROR in ./app/app.js
Module not found: Error: Can't resolve 'C:UsersSomeoneSomewhere
ode_modulesangular-hot-loaderlibangular-hot-loader' in [...]
```

For some reason, the `\` in windows path were acting as escape characters. After some research inside your code, I found out that this was because you are using a string template in `loader.js` around line 39:

```js
prependText = `
    if (module.hot) {
      module.hot.accept();
      var hotAngular = require('${require.resolve('./angular-hot-loader')}')(${JSON.stringify(config)});
    }
  `;
```

So basically `require.resolve(...)` resolves the path correctly (i.e. with windows' `\`), but then once inserted in the string template as a string (inside quotes `'`), all the `\` become escape characters and screw things up.
Changing the previous piece of code by this one:

```js
var resolved = require.resolve('./angular-hot-loader').replace(/\\/g, '\\\\');

  prependText = `
    if (module.hot) {
      module.hot.accept();
      var hotAngular = require('${resolved}')(${JSON.stringify(config)});
    }
  `;
```
therefore solved the problem.

If you're OK with this I'd like to open a pull request with the fix above.
Thanks !

**Comment by vitaliy-bobrov** [comment]
@Sn0wFox ,
Hi, I'm ok with replacing slashes in path. As for HMR for Anuglar 2+ - it is already implemented, for example look into angular-cli project.
